components/widgets/resources/src/main/resources/resources/uicomponents/widgets/ageOfOnset.js

```diff
@@ -0,0 +1,100 @@
+var PhenoTips = (function(PhenoTips) {
+  // Start PhenoTips augmentation
+  var widgets = PhenoTips.widgets = PhenoTips.widgets || {};
+
+  widgets.AgeOfOnset = Class.create({
+    // The age until which the specified HPO term applies
+    ageMonthsToHpoTerm: [
+      [0.92008, "HP:0003623"],
```

Comment:
28 days == 0.92008 months. Maybe this should be expressed more clearly by including the math inline, or using an explanation in a comment.